mw-webapp/src/index.tsx

```diff
@@ -14,12 +15,14 @@ const root = ReactDOM.createRoot(
 root.render(
   <React.StrictMode>
     <BrowserRouter>
-      <App />
+      <Theme>
```

Comment:
why do we need it?

Reply:
This is the third point install Radix Themes.

[Installation](https://www.radix-ui.com/themes/docs/overview/getting-started#installation)
[1. Install Radix Themes](https://www.radix-ui.com/themes/docs/overview/getting-started#1-install-radix-themes)
[2. Import the CSS file](https://www.radix-ui.com/themes/docs/overview/getting-started#2-import-the-css-file)
[3. Add the Theme component](https://www.radix-ui.com/themes/docs/overview/getting-started#3-add-the-theme-component)

Reply:
Yeah, I saw it. Sorry, I will specify my question: does it bring any additional styles to the project? If yes - we currently trying to avoid unnecessary styles to make our own style system in the near future, in this case, we don't need Radix styles, agreed?

Reply:
I agree) removed)